docs/content/reference/command-line-interface/_index.md

```diff
@@ -462,6 +536,64 @@ drasi-system
 **Known Issues**: 
 - The `namespace` command does not currently enforce any restrictions on namespace names, nor does it validate that the namespace used in the `namespace set` command exist. Ensure that the namespace names used do not conflict with existing namespaces or reserved names.
 
+
+### drasi secret
+**Purpose**: The `secret` command provides a subset of commands for managing secrets in the default secret store of the hosting platform. In the case of Kubernetes, this will be Kubernetes secrets in the `drasi-system` namespace. These can then be referenced when creating Drasi resources, such as Sources or Reactions with confidential connection information. Secret values are stored under keys within a named secret.
```

Comment:
If we installed Drasi to a different namespace, will `drasi secret set` create the secrets in the new namespace?

Reply:
Yes, we have the [PlatformClient interface](https://github.com/drasi-project/drasi-platform/blob/main/cli/sdk/platform_client.go#L29), which is an abstraction around the hosting platform. It has a "SetSecret" method, and in the Kubernetes implementation, we use the namespace specified in the config: https://github.com/drasi-project/drasi-platform/blob/main/cli/sdk/platform_client.go#L193